packages/module/src/LogViewer/LogViewer.tsx

```diff
@@ -168,6 +178,9 @@ const LogViewerBase: React.FunctionComponent<LogViewerProps> = memo(
     }, [parsedData, scrollToRow]);
 
     const createDummyElements = () => {
+      if (!(containerRef && containerRef.current)) {
+        return;
+      }
```

Comment:
To prevent manipulation of DOM when the timer callback happens after the container was closed.